packages/.vscode/settings.json

```diff
@@ -2,4 +2,8 @@
     // Set current working directory to devtools_app.
     "terminal.integrated.cwd": "devtools_app",
     "dart.showTodos": false,
+    "editor.codeActionsOnSave": {
+        "source.fixAll": true,
+        "source.dcm.fixAll": true
```

Comment:
Do we really trust these enough to automatically apply fixes on save? I'd worry that fixes for false positives would slip in and potentially cause issues.

Reply:
Removed.